src/main/java/seedu/address/model/event/Event.java

```diff
@@ -1,11 +1,149 @@
 package seedu.address.model.event;
 
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.member.Member;
+
 /**
  * Represents an Event in the address book.
  * Guarantees: details are present and not null, field values are validated, immutable.
  */
 public class Event {
-    public Event(){
+    // Identity fields
+    private final String name;
+    private final LocalDate date;
+
+    // Data fields
+    private final HashMap<Member, Boolean> participants = new HashMap<>();
+
+    /**
+     * Every field must be present and not null.
```

Comment:
Try following the javadocs for constructors in the tp?

This is taken from UiManager class for your reference.
/**
 * Creates a {@code UiManager} with the given {@code Logic}.
 */
public UiManager {...}